static/index.html

```diff
@@ -30,12 +30,14 @@
 <body>
 <div layout="row" id="wrapper" ng-controller="AppCtrl" layout="column" ng-cloak>
     <md-sidenav class="md-sidenav-left" md-component-id="left" md-is-locked-open="$mdMedia('gt-md')" md-disable-backdrop md-whiteframe="4">
-	  <header class="nav-header">
-        <a ng-href="/#/" class="docs-logo">
-          <img src="images/logo.svg" alt="" />
-          <span class="docs-logotype">Honeypot</span>
-        </a>
-	  </header>
+	  <a ng-click="href('/'); toggleLeft()">
```

Comment:
We need ng-click to be able to run the toggleLeft()-function afterwards, which closes the menu in mobile view.